components/budget/MissingBudget.tsx

```diff
@@ -11,9 +11,10 @@ export interface MissingBudgetProps {
     passwordHash: string;
     month: string;
     year: number;
+    navigation: (budgetCategory: Budget) => void;
```

Comment:
Use the useNavigation hook and call navigation.navigate in the component instead of taking a navigation prop.

Reply:
Also, since this is a callback, it should have followed the callback naming convention. Eg: onSubmit